sys/net/network_layer/icmpv6/icmpv6_hdr_print.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_pktdump
+ * @{
+ *
+ * @file
+ * @brief       Generic module to dump packages received via netapi to STDOUT
```

Comment:
C-files don't need a `@brief` (it's not parsed by doxygen anyways) and the description is so generic I wouldn't even see it as internal doc.